src/main/java/Giaithua.java

```diff
@@ -0,0 +1,11 @@
+public class Giaithua {
+    public long giaithua(int n) {
+        long gt = 1;
+        int i = 1;
+        while (i<=n) {
```

Comment:
Dĩ nhiên là dùng `for` hay `while` thì cũng được việc cả, nhưng trong những tình huống như thế này, vòng for bao giờ cũng dễ hiểu hơn vòng while